extensions/include/cmark-gfm-core-extensions.h

```diff
@@ -1,16 +1,17 @@
 #ifndef CMARK_GFM_CORE_EXTENSIONS_H
 #define CMARK_GFM_CORE_EXTENSIONS_H
 
+#include "cmark-gfm-extension_api.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-#include "cmark-gfm-extension_api.h"
 #include "export.h"
```

Comment:
Should we just move all the header includes out of the `extern "C"` block? I guess i'm not sure what the precise semantics are that Clang is complaining about, so i'm not sure why this header include is fine while the others are not.

It looks like the `module.modulemap` file for the base cmark-gfm library doesn't include this header, which is likely why it's not considered a problem. What's stopping us from adding `export.h` to that module map?

Reply:
I just moved the smallest set that the compiler complained about, but I think you're right. I'll try adding export.h to the module map and see what happens, and then move that header outside of the block as well.

Reply:
Yep! Adding `export.h` to the base cmark-gfm `module.modulemap` incurred the same error. I've updated the PR with this inclusion to the modulemap, and moving the include for `export.h` outside the `extern "C"` block as well.